model/database.php

```diff
@@ -37,28 +36,28 @@ function __construct()
      */
     function getTutors($year = '2020')
     {
-            //defining query
-            $sql = "SELECT Tutor.tutor_first, Tutor.tutor_last, User.user_email, Year.year_packet_sent, Year.year_background,
+        //defining query
+        $sql = "SELECT Year.year_id, Tutor.tutor_first, Tutor.tutor_last, User.user_email, Year.year_packet_sent, Year.year_background,
                     Year.year_reference, Year.year_offer_letter, Year.year_affirmation_disclosures, Year.year_sexual_misconduct,
                     Year.year_w4, Year.year_handbook_verification, Year.year_ADP, Year.year_i9, Year.year_orientation,
                     Year.year_placement from Year
                     JOIN Tutor on Year.user_id = Tutor.user_id
                     JOIN User on Year.user_id = User.user_id
                     where Year.year_start = :year";
 
-            //Preparing statement
-            $statement = $this->_dbh->prepare($sql);
+        //Preparing statement
+        $statement = $this->_dbh->prepare($sql);
 
-            //Binding Parameters
-            $statement->bindParam(':year', $year);
+        //Binding Parameters
+        $statement->bindParam(':year', $year);
```

Comment:
We should stick to bind param or pass in the variables into the execute method. Unless we decide to use both.